vertx-web-client/src/main/java/io/vertx/ext/web/client/impl/CacheManagerImpl.java

```diff
@@ -0,0 +1,29 @@
+package io.vertx.ext.web.client.impl;
+
+import io.vertx.ext.web.client.CacheManager;
+import io.vertx.ext.web.client.CacheOptions;
+import io.vertx.ext.web.client.WebClientOptions;
+
+/**
+ * @author Alexey Soshin
+ */
+public class CacheManagerImpl implements CacheManager {
+    private CacheInterceptor cacheInterceptor;
```

Comment:
make it final

Reply:
I think making it final will introduce more problems that it will solve. Nowadays it's initialized only if correct options were supplied.